packages/client/src/Dashboard.tsx

```diff
@@ -436,8 +452,63 @@ function ConversationRow({
           secondary: classes.snippetText
         }}
       />
+      {newLabels &&
+        newLabels.map(label => (
+          <DisplayLabel key={labelId + count++} label={label} />
```

Comment:
I don't like the use of a `count` variable that is scoped over the entire component. If you want an index for each element when iterating over an array then use the second parameter of the `map` callback:

```ts
newLabels.map((label, index) => (
  <DisplayLabel key={labelId + index} label={label} />
))
```

But using the array index as the `key` prop should be a last resort - it's better to use a key that is intrinsically linked to the thing that is being displayed. In this case the key could be the value of the `label` variable itself.

It looks like you incorporated the conversation id into the key to avoid collisions with keys in other conversation rows. That isn't necessary: a `key` prop only needs to be unique within the same list. So as long as the conversation does not have two labels with the same text then you can use the label text by itself as the key.